regolith/helpers/l_generalhelper.py

```diff
@@ -14,19 +14,22 @@
 def subparser(subpi):
     subpi.add_argument(
         "coll",
-        help='collection where the lister is going to run.')
+        help='Collection where the lister is going to run.')
     subpi.add_argument(
         "-f","--kv_filter",
         nargs="+",
-        help='search the given collection by key element pairs')
+        help="Search the given collection by key element pairs. "
```

Comment:
"key-value pairs".  We don't call those things "elements"

Reply:
done!